kubernetes/overlays/prod-butler/ingestd.yml

```diff
@@ -0,0 +1,10 @@
+brokers: 134.79.23.221:9094
+group_id: "prod_ingestd"
+num_messages: 50
+butler:
+  instrument: lsst.obs.subaru.HyperSuprimeCam
+  repo: /sdf/data/rubin/repo/hsc_pdr2_multisite
```

Comment:
This is so that we can have a repo (and corresponding Rucio scope) per "campaign".  Trying to merge the repo and a collection into a scope seemed difficult.  But there are indeed other ways we could do this.

Reply:
The other way around; the Rucio scope is used as the repo name.